app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -103,6 +104,12 @@ class EventsFragment : Fragment() {
             showNoInternetScreen(isNetworkConnected)
         }
 
+        rootView.swiperefresh.setColorSchemeColors(Color.BLUE)
+        rootView.swiperefresh.setOnRefreshListener({
+            eventsViewModel.loadLocationEvents(eventsViewModel.savedLocation.toString())
+            rootView.swiperefresh.isRefreshing = false
```

Comment:
Spinner should disable once items are done loading

Reply:
I don't know why this is not happening. I can see in the logs that the events are being fetched but spinner keeps loading infinitely if I remove this

Reply:
You don't have to remove this, you have to bind it to a LiveData which will push true once it starts loading and false once loading is done